src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -179,6 +179,7 @@ public async Task<Either<ActionResult, PaginatedListViewModel<PublicationSearchR
                 {
                     Id = result.Value.Id,
                     Slug = result.Value.Slug,
+                    ReleaseSlug = result.Value.LatestPublishedReleaseVersion!.Release.Slug,
```

Comment:
I think there's tests in `Content.Services.Tests.PublicationServiceTests` which could be updated to test this.

Also there's `GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Controllers.PublicationControllerCachingTests._publications` which should be updated to include this value.